internal/cmd/gocdk/launch.go

```diff
@@ -15,11 +15,198 @@
 package main
 
 import (
+	"bytes"
 	"context"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"log"
+	"net/url"
+	"os/exec"
+	"path/filepath"
+	"strconv"
 
 	"golang.org/x/xerrors"
 )
 
 func launch(ctx context.Context, pctx *processContext, args []string) error {
-	return xerrors.New("not implemented")
+	f := newFlagSet(pctx, "launch")
+	dockerImage := f.String("image", "", "Docker image to launch in the form `image[:tag]` (defaults to project latest)")
```

Comment:
The dockermentation uses `name:tag`, and I think we should too. (Not objecting to the optional brackets, just to calling it "image" instead of "name".)

Reply:
Done.